### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by bpuckhaber**

### CODE REVIEW

**Review by wendyperalta — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 [comment]: <> "NOTE! Ensure all images are added via the \[label\]\(link\) syntax!"
 
+## 2023-12-13
+| What's new | |
+| :--------- | :-: |
+| The Insights **Clients filter** now has the ability to **search by client name and version**. It has also found a new home alongside the operations and fields filters. | <img width="1439" alt="changelog" src="https://github.com/apollographql/apollo-studio-community/assets/9868979/2fd9bb5a-63ce-4235-8d9b-70b9bc0aef20"> |
```
Comment (Meschreiber) [inline CHANGELOG.md:6]:
```suggestion
| The Insights **Clients filter** now can **search by client name and version**. The filter has found a new home to the right of the operations and fields search bar. | <img width="1439" alt="changelog" src="https://github.com/apollographql/apollo-studio-community/assets/9868979/2fd9bb5a-63ce-4235-8d9b-70b9bc0aef20"> |
```

This seems like a slightly clearer marker.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
...
+| Additionally, the Insights page now has **Fast Mode** time range options. These options exclude the last 24 hours of data to help increase load times for large graphs. |![image](https://github.com/apollographql/apollo-studio-community/assets/9286598/ecd233c5-8fac-48db-a0d2-c1f4dce3c3a5) |
```
Comment (Meschreiber) [inline CHANGELOG.md]:
```suggestion
| Additionally, the Insights page now has **Fast Mode** when selecting your metrics' time range. Fast mode excludes the last 24 hours of data to enhance load times for large graphs. |![image](https://github.com/apollographql/apollo-studio-community/assets/9286598/ecd233c5-8fac-48db-a0d2-c1f4dce3c3a5) |
```

**Review by Meschreiber — COMMENTED** [review]
Just some small wording suggestions!

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,10 @@
 [comment]: <> "NOTE! Ensure all images are added via the \[label\]\(link\) syntax!"
 
+## 2023-12-13
+| What's new | |
+| :--------- | :-: |
+| The Insights **Clients filter** now has the ability to **search by client name and version**. It has also found a new home alongside the operations and fields filters. | <img width="1439" alt="changelog" src="https://github.com/apollographql/apollo-studio-community/assets/9868979/2fd9bb5a-63ce-4235-8d9b-70b9bc0aef20"> |
+| Additionally, the Insights page now has **Fast Mode** time range options. These options exclude the last 24 hours of data to help decrease load times for large graphs. |![image](https://github.com/apollographql/apollo-studio-community/assets/9286598/ecd233c5-8fac-48db-a0d2-c1f4dce3c3a5) |
```
Comment (Meschreiber) [inline CHANGELOG.md:7]:
```suggestion
| Additionally, the Insights page now has **Fast Mode** time range options. These options exclude the last 24 hours of data to decrease load times for large graphs. |![image](https://github.com/apollographql/apollo-studio-community/assets/9286598/ecd233c5-8fac-48db-a0d2-c1f4dce3c3a5) |
```

**Review by Meschreiber — APPROVED** [review]
Meant to leave suggestions with an approve ✅

**Pull request merged**